DOCS-3003: Add prerequisite of physically assembling hardware to components and services (#3540)
viamrobotics/docs: docs/components/camera/webcam.md

`webcam` is the general camera model.
If the camera drivers are among those in [this mediadevices repository](https://github.com/pion/mediadevices), the camera will work with the webcam model.

> First, connect your camera to your machine's computer (unless it is inbuilt like a webcam on a laptop) and turn it on.

Reviewer: ```suggestion
First, connect your camera to your machine's computer (unless it is built-in like a webcam on a laptop).
```
[nit] since we are US-based and use the much more common (in the US) "built-in" throughout the rest of the docs
Also usually the camera gets its power from the machine, so you don't do anything to specifically turn the camera on. By "it" do you mean the camera or the machine?

Author: I mean the computer

Reviewer: "it" is ambiguous in this case throughout the PR--please eliminate or edit to clarify throughout.
